core/trino-web-ui/src/main/resources/webapp/vendor/vis/dist/vis.css

```diff
@@ -254,7 +254,7 @@
 }
 
 .vis.timeline .item .delete {
-  background: url('img/timeline/delete.png') no-repeat top center;
+  background: url('dist/img/timeline/delete.png') no-repeat top center;
```

Comment:
i assume the build puts them there since are deleting the files

Reply:
yes